src/utils/report.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Report a violation.
+ *
+ * @param {object} v - Violation details object
+ * @param {string} v.ruleName - The name of the rule
+ * @param {Result} v.result - PostCSS Result object
+ * @param {string} v.message - Message to inform user of the violation
+ * @param {Node} v.node - PostCSS Node object
+ */
+export default function (v) {
```

Comment:
Just weariness of typing ... I should not succumb :) Happy to rename it "violation" or something.



> On Jun 7, 2015, at 10:56 PM, Maxime Thirouin <notifications@github.com> wrote:
> 
> In src/utils/report.js:
> 
> > @@ -0,0 +1,28 @@
> > +/**
> > + * Report a violation.
> > + *
> > + * @param {object} v - Violation details object
> > + * @param {string} v.ruleName - The name of the rule
> > + * @param {Result} v.result - PostCSS Result object
> > + * @param {string} v.message - Message to inform user of the violation
> > + * @param {Node} v.node - PostCSS Node object
> > + */
> > +export default function (v) {
> not a fan of using a single letter object but that's a detail ^^
> 
> —
> Reply to this email directly or view it on GitHub.
>